client/containers/Pub/PubHeader/ReviewHeaderSticky.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react';
+import { Button } from '@blueprintjs/core';
+
+import { usePageContext } from 'utils/hooks';
+import { pubUrl } from 'utils/canonicalUrls';
+import { Icon } from 'components';
+import { apiFetch } from 'client/utils/apiFetch';
+import { useLocalStorage } from 'client/utils/useLocalStorage';
+import { getEmptyDoc } from 'client/components/Editor';
+import { DocJson } from 'types';
+
+import { usePubContext } from '../pubHooks';
+import Review from './Review/Review';
+import ReviewerDialog from './Review/ReviewerDialog';
+
+require('./reviewHeaderSticky.scss');
+
+const ReviewHeaderSticky = () => {
+	const { pubData, updatePubData } = usePubContext();
+	const {
+		communityData,
+		scopeData: { activePermissions, memberData },
+		loginData: { fullName },
+	} = usePageContext();
+
+	const [visible, setVisible] = useState(false);
+	const [reviewTitle, setReviewTitle] = useState('Untitled Review');
+	const [reviewerName, setReviewerName] = useState('');
+	const [createError, setCreateError] = useState(undefined);
+	const [isLoading, setIsLoading] = useState(false);
+	const [createdReview, setCreatedReview] = useState(false);
+	const [showReview, setShowReview] = useState(false);
+	const [reviewNumber, setReviewNumber] = useState(0);
+
+	// creates a docjoson object in local store, provides state handlers as well
+	const { value: review, setValue: setReview } = useLocalStorage<DocJson>({
+		initial: () => getEmptyDoc(),
```

Comment:
Can remove this intermediate arrow function and just pass `getEmptyDoc`

Reply:
I get this error if i do 
<img width="1089" alt="image" src="https://user-images.githubusercontent.com/34730449/187718613-e23b86fc-54d6-4f93-8fc6-ebb5ccabd955.png">

Reply:
i lied its a callback so removing the call works